### PR TITLE
Androapp 5893 Fix Org unit selector bottom sheet height

### PR DESCRIPTION
**Pull request by siddh1004**
Fix includes:

1. Size of bottom sheet when there are less items or `No Results Found`.
2. Fix the alignment of bottom divider

### CODE REVIEW

File: designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/BottomSheet.kt
Comment (andresmr) [inline designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/BottomSheet.kt:1]: Can this BottomSheet be internal?

File: designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/OrgBottomSheet.kt
Comment (andresmr) [inline designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/OrgBottomSheet.kt:1]: What do you think about create a test for the Orgbottomsheet and a snapshot testing as well?

File: designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/BottomSheet.kt
Comment (siddh1004) [reply designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/BottomSheet.kt:1]: @andresmr We can't make this internal, as it's been used in capture app at various places such as `QRDetailBottomDialog`, `DeleteBottomSheetDialog` and `SchedulingDialogUi`.

File: designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/OrgBottomSheet.kt
Comment (siddh1004) [reply designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/OrgBottomSheet.kt:1]: @andresmr There is open bug with papparazi that creates a blank screen when we try to do snapshot testing for ModalBottomSheet

**Review by andresmr — APPROVED** [review]

**Review by xavimolloy — APPROVED** [review]

**Pull request merged**